README.md

```diff
@@ -84,6 +84,35 @@ changelogFile: CHANGELOG.md
 
 When a package is modified in any way (including updates of the context information in the `.janno` file), then the `packageVersion` field SHOULD be incremented and the `lastModified` field updated to the current date.
 
+#### Package versioning
+
+The `packageVersion` field is a mandatory entry of the `POSEIDON.yml` file. It denotes the version of the individual package, using a three-component versioning system derived from [semantic versioning](https://semver.org).
+
+Each version number is comprised of three numbers, separated by a `.`. For example: `0.1.0`, `1.0.0` or `2.1.3`. The first number gives the `major`, the second the `minor` and the third the `patch` component of the version number. For a Poseidon package these components SHOULD be incremented, so raised by one, when the following changes occur:
+
+- **`major`** (first number)
+  - When samples are added to a package.
+  - When samples are removed from a package.
+  - When the genotype data (i.e. the contents of the `.bed`/`.bim`/`.fam` or `.geno`/`.snp`/`.ind` files) for any number of samples is changed.
+
+- **`minor`** (second number)
+  - When larger pieces of meta- or context information are added or modified in any package file, except the genotype data. For example:
+    - An entire `.janno`, `.bib` or `.ssf` file is added or replaced.
+    - Entire columns in the `.janno` or `.ssf` file are added or replaced.
+    - Primary publications for samples in the `.janno` and `.bib` file are added or replaced.
+
+- **`patch`** (third number)
+  - When smaller pieces of meta- or context information are added or modified in any package file, except the genotype data. For example:
+    - Individual entries in the `.janno` or `.ssf` file are added or replaced.
+    - Secondary publications for samples in the `.janno` and `.bib` file are added or replaced.
+    - BibTeX entries in the `.bib` file are modified.
+    - The package `description` changes in the `POSEIDON.yml` file.
+    - The `CHANGELOG.md` file is modified with additional information on previous entries.
+
+When the `packageVersion` is changed, then the `lastModified` date MUST be updated and an entry to the `CHANGELOG.md` file SHOULD be added.
+
+Packages SHOULD start at `packageVersion` `0.1.0`.
```

Comment:
i.e. that is the version we want packages to have when submitted to the archives, or is that to say that `trident init` creates the package with that version, and `trident validate` will complain with anything below that?

Reply:
None of that. The archive and the tooling have no place in the schema specification. The community-archive actually requires a stable package version, so specifically `1.0.0`, but that is irrelevant here.

`0.1.0` is just the base version that should be used when one starts to develop a package.